openapi.yaml

```
@@ -8,6 +8,79 @@ info:
     email: mail@kopfsachen.org
     url: 'https://www.kopfsachen.org'
 paths:
+  '/self-service/registration/api':
+    get:
+      summary: Initate registration flow
+      tags:
+        - authentication
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/registration'
+        '500':
+          description: Internal Server Error
+  '/self-service/registration/flow={id}':
```

```suggestion
  '/self-service/registration':
```

Usually not written like that, also query parameters are appended with '?' Same for other paths